packages/eslint-plugin/docs/rules/prefer-ts-expect-error.md

```diff
@@ -20,6 +20,16 @@ Examples of **incorrect** code for this rule:
 // @ts-ignore
 const str: string = 1;
 
+{
+  /* @ts-ignore */
+}
+const value: number = 'value';
+
+/**
+ * @ts-ignore
+ */
+const block: string = 1;
```

Comment:
this is not valid code - the expect error does not apply to the next line in either of these cases
[repl](https://www.typescriptlang.org/play?#code/N4KABGD0BUYAIBcDOBaAlgcwHYHsBOApmNJCAL4gDGOWSCYAbgIYA2ArgQFxhZsC2AIwJ4wAXjAByZuwISA3CBjRwsRKky5CK0tVr0BLHJQDW3OnjRYMYsAEY5QA)

Reply:
Correct now